src/OXRS_WT32.cpp

```diff
@@ -47,6 +47,9 @@ DynamicJsonDocument _fwCommandSchema(JSON_CONFIG_MAX_SIZE);
 jsonCallback _onConfig;
 jsonCallback _onCommand;
 
+// local variables
+char _fwVersion[40] = "";
```

Comment:
Worth defaulting it to something here so if the firmware forgets to set the version it displays something alerting you to the fact? E.g. maybe `<No Version>`?